bindings/wasm/src/account/generation.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use wasm_bindgen::prelude::*;
+
+use identity::account::Generation as Generation_;
+
+use crate::error::Result;
+use crate::error::WasmResult;
+
+#[wasm_bindgen(js_name = Generation, inspectable)]
+#[derive(Clone, Copy, Default, Hash, PartialEq, Eq, PartialOrd, Ord, Deserialize, Serialize)]
+pub struct WasmGeneration(pub(crate) Generation_);
```

Comment:
Aside: do we actually need all the derived traits on Wasm wrappers? I don't think we actually use them anywhere so this could just be code bloat?